src/main.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
+import './index.css';
+import ErrorBoundary from './components/ErrorBoundary/ErrorBoundary';
+
+ReactDOM.createRoot(document.getElementById('root')!).render(
+  <React.StrictMode>
+    <ErrorBoundary
```

Comment:
Better, move error boundary from main.tsx to nested components

Reply:
Fixed in https://github.com/ylakhtin/rs-react-components/commit/273c5a09d9eff0de15d710b09ad3891cb79ee348